pkg/gitreceive/run.go

```diff
@@ -48,9 +50,16 @@ func Run(conf *Config, fs sys.FS, env sys.Env, storageDriver storagedriver.Stora
 
 		log.Debug("read [%s,%s,%s]", oldRev, newRev, refName)
 
-		if err := receive(conf, builderKey, newRev); err != nil {
+		client, err := controller.New()
```

Comment:
Fixed! `build()` now takes a parameter for a `deis.Client`. I pass the client created in run to the build method.